conda-recipe/meta.yaml

```diff
@@ -1,56 +1,60 @@
+# NOTE: This script has been automatically generated by github.com/ericdill/conda-skeletor
+
 package:
   name: metadatastore
   version: {{ environ['GIT_DESCRIBE_TAG'] }}.post{{ environ['GIT_DESCRIBE_NUMBER'] }}
 
 source:
   git_url: ../
+
   patches:
     - config.patch
-
 build:
   number: 0
 
-  # Note that this will override the default build string with the Python
-  # and NumPy versions
   string: {{ environ.get('GIT_BUILD_STR', '') }}_py{{ py }}
 
+  # TODO add entry points
 
 requirements:
   build:
     - python
-    - six
-
   run:
     - python
-    - mongoengine ==0.8.7
-    - pymongo ==2.9
-    - six
-    - pyyaml
-    - prettytable
-    - humanize
-    - numpy
-    - pytz
-    - jinja2
     - boltons
+    - mongoengine
+    - pymongo <3
+    - pytz
+    - pyyaml
+    - six
 
 test:
   requires:
-    - nslsii_dev_configuration
+    - boltons
+    - coverage
+    - mongoengine
```

Comment:
Probably.  Also I recently discovered that test requires is a superset of run requires, so much of this section can just be deleted